network/gui-ufw/path-fix.diff

```diff
@@ -0,0 +1,16 @@
+--- a/bin/gufw-pkexec
++++ b/bin/gufw-pkexec
+@@ -1,2 +1,11 @@
+ #!/bin/bash
+-python3 /usr/share/gufw/gufw/gufw.py $1
++LOCATIONS=`ls -ld /usr/lib64/python*/site-packages/gufw/gufw.py | awk '{print $9}'` # from source
```

Comment:
this one is spesific for x86_64?

Reply:
I hadn't even noticed I had done that. I'll have to test on 32bit, there's nothing actually compiling  so it should work on any platform.

Reply:
I pushed a small edit to the patch to fix on 32bit. Tested & verified working.